client/src/api/Rank/rank_crud.ts

```diff
@@ -0,0 +1,35 @@
+import {
+	TopPostsRes,
+	TopCommentsRes,
+	TopActivitiesRes,
+} from "../../../../nestjs-migration/dist/rank/dto/rank-results.dto";
+
```

Comment:
@IIINHWAA 
`nit` 이후에 리팩터링하실 때, .env파일로 API_URL을 관리하는게 좋을 것 같습니다.

```suggestion
// .env 파일에 API_URL 추가
REACT_APP_API_URL=http://localhost:3000

// 코드에서 사용
const API_URL = process.env.REACT_APP_API_URL;

const response = await fetch(`${API_URL}/api/rank/posts`);
```